cmd/tailscale/cli/up.go

```diff
@@ -609,6 +664,16 @@ func runUp(ctx context.Context, args []string) error {
 	}
 }
 
+func printUpDoneJSON(state, errorString string) {
```

Comment:
change `state` from type `string` to `ipn.State`?

Reply:
Done.